timescaledb/how-to-guides/hypertables/distributed-hypertables.md

```diff
@@ -71,6 +81,91 @@ correct data node in a stream. `COPY` usually delivers better performance than
 `INSERT`, although it doesn't support features like conflict handling (`ON
 CONFLICT` clause) that are used for [upserts][upserts].
 
+## Using triggers on distributed hypertables
+Triggers on distributed hypertables work similar to triggers on
+regular hypertables, including having the same limitations. However,
+due to data and tables being distributed across many data nodes, there
+are some notable differences compared to regular hypertables:
+
+* Row-level triggers fire on the data node where a row is inserted
+* Statement-level triggers fire once on each affected node,
+  including the access node
+* A replication factor greater than 1 further increases the number of
+  nodes that a trigger fires on (i.e., each replica node fires the
+  trigger)
+
+### Creating triggers
+A trigger is created on distributed hypertables with [`CREATE
+TRIGGER`][create-trigger] as normal. The trigger is created on each
+data node, including the function that the trigger executes. However,
+any other functions or objects referenced in the trigger need to be
+present on all nodes prior to creating the trigger. To create a
+function or other referenced object on all data nodes, use the
+[`distributed_exec`][dist_exec] procedure.
+
+
+```sql
+-- Create the trigger function on the access node
+CREATE OR REPLACE FUNCTION my_trigger_func()
+	RETURNS TRIGGER LANGUAGE PLPGSQL AS
+$BODY$
+BEGIN
+	RAISE NOTICE 'trigger fired';
+	RETURN NEW;
+END
+$BODY$;
+
+-- Create the trigger
+CREATE TRIGGER my_trigger
+	AFTER INSERT ON hyper
+	FOR EACH ROW
+	EXECUTE FUNCTION my_trigger_func();
+```
```

Comment:
It would be better if you could split these up, and keep the comments out of the text box.